examples/front-proxy/docker-compose.yaml

```
@@ -17,6 +17,8 @@ services:
       - "8080:8080"
       - "8443:8443"
       - "8001:8001"
+    environment:
+      - ENVOY_UID=0
```

```suggestion
      - ENVOY_UID
```

we can do this so it propagates from your local env, while the default is still non-root?